### PR TITLE
Product Search Dropdown

### PR DESCRIPTION
**Pull request by harryadelb**
Hello this PR is intended for #154 where we add search functionality for the products via a search dropdown.

P.S. Check your mail :wink:

### CODE REVIEW

File: examples/controlpanel/pages/products/index.jsx [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-export default connectApollo((props) => (
-  <App {...props}>
-    <Container>
-      <h2>Products</h2>
-      <ProductList />
-    </Container>
-  </App>
-));
+export default connectApollo(
+  withRouter(({ router, ...rest }) => {
```
Comment (schmidsi) [inline examples/controlpanel/pages/products/index.jsx:9]: Would be better to put `withRouter` inside the compose call of ProductList

File: examples/minimal/boot.js [hunk 2]
```diff
@@ -40,10 +40,12 @@ const initializeDatabase = () => {
     if (Users.find({ username: 'admin' }).count() > 0) {
       return;
     }
+
     const admin = Factory.create('user', {
       username: 'admin',
       roles: ['admin'],
       emails: [{ address: 'admin@localhost', verified: true }],
+      profile: { address: {} },
```
[the remark this answers is not in the document]
Comment (harryadelb) [reply examples/minimal/boot.js:48]: Thanks!

File: examples/controlpanel/pages/products/index.jsx (hunk 1 above)
Comment (harryadelb) [reply examples/controlpanel/pages/products/index.jsx]: That should be done, please recheck.

**Comment by pozylon** [comment]
@harryadelb Thanks for contributing, we will check our mail :)

File: examples/controlpanel/package.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "prepublishOnly": "npm run export"
   },
   "dependencies": {
+    "lodash": "^4.17.15",
```
Comment (pozylon) [inline examples/controlpanel/package.json:30]: As publish the npm package as pre-bundled, all dependencies have to go into "devDependencies"

**Pull request merged**